### PR TITLE
Plot_graph.py added to Other Algorithms

### PR DESCRIPTION
**Pull request by charlie219**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

YES

### Description

As assigned by @HarshCasper for issue #1745, I have added the code for plotting various function in the graph using the matplotlib library of python.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#1745 

I request @iamrajiv, @HarshCasper and @atarax665 to review this PR. Thank you 👍

### CODE REVIEW

File: Python/other/Plot_Graph.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+#f = lambda x : rand.randint(0,100)
+#f = lambda x : mt.sqrt(x)
+#f = lambda x: 1/x
+
+
+# Number of Y co-ordinates
+limit = 50
+
+# Calculation the X co-ordinated using the function declared above
+for i in range(-limit, limit):
```
Comment (atarax665) [inline Python/other/Plot_Graph.py:41]: What if I try to plot a graph of a function of form ```f(x) = 1 / x^n```? I guess the loop won't be able to append the value of y coordinate when the value of x gets 0.

**Review by atarax665 — CHANGES_REQUESTED** [review]

File: Python/other/Plot_Graph.py (hunk 1 above)
Comment (charlie219) [reply Python/other/Plot_Graph.py:41]: that's why under the loop I have given an `if` statement. If the user chooses  `f(x)=1/x` function, then he has to modify the loop accordingly. Same is true for the trigonometric functions, the user has to modify the limits.

Comment (atarax665) [reply Python/other/Plot_Graph.py:41]: Okay, so please add a proper NOTE regarding this to avoid any such confusion.

**Review by atarax665 — APPROVED** [review]

**Review by iamrajiv — APPROVED** [review]

File: Python/other/Plot_Graph.py (hunk 1 above)
Comment (charlie219) [reply Python/other/Plot_Graph.py:41]: Sure

**Comment by charlie219** [comment]
@atarax665 @iamrajiv  the `NOTE` is added, please review

**Review by iamrajiv — APPROVED** [review]

**Review by atarax665 — APPROVED** [review]

**Pull request merged**